qbittorrent/umbrel-app.yml

```diff
@@ -0,0 +1,30 @@
+manifestVersion: 1
+id: qbittorrent
+category: Networking
+name: qBittorrent
+version: "4.5.2"
+tagline: A fast, easy and free BitTorrent client
+description: >-
+  The qBittorrent project aims to provide an open-source software alternative to µTorrent.
+  
+  Additionally, qBittorrent runs and provides the same features on all major platforms (FreeBSD, Linux, macOS, OS/2, Windows).
+  
+  Bittorrent is based on the Qt toolkit and libtorrent-rasterbar library.
+  
+  ⚠️ qBittorrent downloads torrents over the Clearnet, not Tor.
```

Comment:
For the description section, a single blank newline between paragraphs places text on their own lines, like this:
<img width="847" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/2a991311-8dea-413d-95cc-120313963190">

but **two** blank newlines are needed between paragraphs to separate the lines with whitespace.

```
  The qBittorrent project aims to provide an open-source software alternative to µTorrent.
  <newline here>
  <newline here>
  Additionally, qBittorrent runs and provides the same features on all major platforms (FreeBSD, Linux, macOS, OS/2, Windows).
```